src/AppConfig.ts

```diff
@@ -28,3 +32,25 @@ export const redisConfig = {
   port: get('redisConfig.port'),
   host: get('redisConfig.host'),
 };
+
+export const emailData = {
+  email: get('emailConfig.email'),
+  password: get('emailConfig.password'),
+  host: get('emailConfig.host'),
+  fromMessage: get('emailConfig.fromMessage'),
+  confirmUrl: get('emailConfig.confirmUrl'),
+  confirmMessageSubject: get('emailConfig.confirmMessageSubject'),
+  confirmMessageText: get('emailConfig.confirmMessageText'),
```

Comment:
Email subjects and texts should be in constants, not in config, in config you should store only data that can be different in production and development versions of the app.(e.g different API keys, urls, secrets and so on)